hydra/grammar/OverrideLexer.g4

```diff
@@ -69,4 +71,17 @@ QUOTED_VALUE:
       '\'' ('\\\''|.)*? '\'' // Single quotes, can contain escaped single quote : /'
     | '"' ('\\"'|.)*? '"' ;  // Double quotes, can contain escaped double quote : /"
 
-INTERPOLATION: '${' ~('}')+ '}';
+////////////////////////
+// INTERPOLATION_MODE //
+////////////////////////
+
+mode INTERPOLATION_MODE;
+
+NESTED_INTER_OPEN: INTER_OPEN -> type(INTER_OPEN), pushMode(INTERPOLATION_MODE);
```

Comment:
not sure if this token type is needed.
looks like it's exactly like INTER_OPEN.
Is this because we need to handle all possible tokens in the mode?

Reply:
All modes are entirely independent, if you want to re-use a rule from another mode it has to be declared again (under a different name)